catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/CrossPlatformFilePathEvaluatorTest.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.ddf.commands.catalog;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.codice.ddf.commands.util.CrossPlatformFilePathEvaluator;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class CrossPlatformFilePathEvaluatorTest {
```

Comment:
might want to add test cases when the file is not found anywhere. If it is found in 2 places (e.g. down/load/a and download/a), tests with Linux files and Windows files.

We might also want to make sure that these work on both Linux/Mac and Windows which might require creating tests cases with assumptions for Windows and for Linux/Mac only.